search-app/src/cmr/search/api/concepts_search.clj

```diff
@@ -366,13 +367,36 @@
   ;; no errors, return 204
   {:status 204})
 
+(defn pluralize-concept
+  "Return the plural form of a concept"
+  [concept]
+  (let [concept-name (name concept) last-letter (subs concept-name (- (count concept-name) 1))]
+    (if (= "y" last-letter)
```

Comment:
so I think this code exists:
(inflections.core/plural "gridy") -> gridies
(inflections.core/singular "gridies") -> gridy

Reply:
see https://cljdoc.org/d/inflections/inflections/0.13.1/api/inflections.core#plural